### DISCUSSION TITLE
Improve preset documentation

### DISCUSSION THREAD

**Discussion by mernst** [renovatebot/renovate#21524] [Suggest an Idea]
### Type of discussion.

I just want to chat

### Tell us more.

https://docs.renovatebot.com/key-concepts/presets/ does not say how to *use* a preset.  Maybe I was missing something obvious, but it took me too long to figure out that one uses a preset by putting it in the `"extends":` part of a Renovate JSON file.  I suggest improving that documentation.

**Comment by viceice** [comment]
feel free to open a PR to extend the docs 🤗

**Reply by HonkingGoose** [reply]
I created a PR for us, see:

- #21534
